packages/katello/katello/hostname-change.rb

```diff
@@ -345,13 +501,17 @@ def run
       STDOUT.puts "updating hostname in foreman installer scenarios"
       self.run_cmd("sed -i -e 's/#{@old_hostname}/#{@new_hostname}/g' #{scenarios_path}/*.yaml")
 
+      STDOUT.puts "updating hostname in hammer configuration"
+      self.run_cmd("sed -i.bak -e 's/#{@old_hostname}/#{@new_hostname}/g' #{hammer_root_config_path}/*.yml")
+      self.run_cmd("sed -i.bak -e 's/#{@old_hostname}/#{@new_hostname}/g' #{hammer_config_path}/*.yml")
+
       if File.exist?(last_scenario_yaml)
         STDOUT.puts 'backing up last_scenario.yaml'
-        temp_last_scenario_yaml = Tempfile.new('last_scenario')
+        @temp_last_scenario_yaml = Tempfile.new('last_scenario')
```

Comment:
There's an attr_accessor defined for @temp_last_scenario_yaml so we should use that unless there was a reason to switch back to the instance variable

Reply:
@jturel I'm used to using `attr_accessor` to access instance variables from outside the class, not from within it.  Tbh I didn't even realize why it was there until this comment.

Reply:
I think I added it a while back when I was making changes in here. I like to avoid referencing instance variables whenever possible. They can be too easily changed and break things especially when they are used across several methods.  Referencing something like a method that we can easily change the implementation of is better